### ISSUE TITLE
Galaxy 7 no can't add face

### ISSUE THREAD

**Issue by timbiotic** [trupici/G-Watch-Wear#43]
I upgraded from 4 to 7 and restored but g-wear watch face is missing now. When I go to install on play store it doesn't show my new watch as option just the old 4. Do I need to start from scratch instead of restoring or has it not been added to 7 yet?

**Comment by trupici** [comment]
Hi,

I haven't tried it on wear os 5 yet. I don't have gw7 and the development
is frozen, sorry. Please follow the posts in Gwatch App Users group, what
other people says.
I would start the installation from scratch, but some people complain that
gwatch wear app is not available in play store on gw7.

Juraj

Dňa so 20. 7. 2024, 14:47 timbiotic ***@***.***> napísal(a):

**Comment by aambrinac** [comment]
Hi
I just upgraded from galaxy watch 4 to ultra and I have the same issue. 
I installed the G Watch Wear app to the watch. In Play Store it is visible as installed on the device. However, no watch face is available to be applied. Also no complication is available to applied to other watch faces.
My guess is that watch face compatibility with Wear OS 6 is the issue.
Alen

**Comment by trupici** [comment]
Yes, due to changes in Wear OS 5 gwatch faces are not supported /
functional. Many users confirmed it.  According to the Samsung docs it
affects not only gwatch apps but the most of watch faces out there.

Dňa št 1. 8. 2024, 14:55 aambrinac ***@***.***> napísal(a):
